src/nft/components/explore/Table.tsx

```diff
@@ -19,6 +19,11 @@ const CELL_WIDTH = '160px'
 const MOBILE_CELL_WIDTH = '240px'
 const DESKTOP_CELL_WIDTH = '360px'
 
+const GlowEffect = styled.div`
+  border-radius: 32px;
```

Comment:
Not sure if it's worth documenting, but generally, I applied a generous border radius to make sure it renders correctly when wrapping a component with border radius.